padertorch/summary/tbx_utils.py

```diff
@@ -139,8 +139,8 @@ def audio(signal, sampling_rate: int = 16000, batch_first=False,
 
     # Normalize so that there is no clipping
     if normalize:
-        signal /= np.max(np.abs(signal))
-        signal *= 0.95
+        signal = signal / np.max(np.abs(signal))
+        signal = signal * 0.95
```

Comment:
Can you change this back to an inplace operation? Only the first shouldn't be inplace. For the second operation you can use inplace